README.md

```diff
@@ -103,5 +103,6 @@ Alexey Strokach,
 Sophy Sun,
 Utku Egemen Umut,
 Sarah Wang,
+Aina Fatema Asim Merchant
```

Comment:
please maintain the alphabetical order (also, make sure to add a comma after your name)